pkg/kubectl/cmd/util/helpers_test.go

```diff
@@ -137,7 +100,7 @@ func TestMerge(t *testing.T) {
 			obj: &api.Service{
 				Spec: api.ServiceSpec{},
 			},
-			fragment: `{ "apiVersion": "v1beta1", "port": 0 }`,
+			fragment: `{ "apiVersion": "v1beta3", "port": 0 }`,
```

Comment:
There's no port field there in v1beta3. Firstly, there's a spec. Secondly, ports are an array.